### ISSUE TITLE
Separate plots from computations

### ISSUE THREAD

**Issue by DominiqueMakowski** [tszanalytics/BayesTesting.jl#5]
Related to #4, having a "structure" for BayesTests would allow separating the plotting aspect: instead of creating a plot and returning it automatically, we could isolate the code and create methods for `plot(x::BayesTest{BayesTestCorrelation})`.

**Comment by tszanalytics** [comment]
Yes, the plotting ecosystem seems to be a bit of mess since v1.0 also, so separating them out makes a lot of sense.  Unfortunately I have almost no experience with object oriented programming, so I have avoided that up to this point.  I think it's a great idea, so if you have a good example I could look at, or want to have a go at a simple version, that would help a lot.
